go.mod

```
@@ -93,6 +93,7 @@ require (
 )
 
 replace github.com/mattn/go-ieproxy => github.com/mattn/go-ieproxy v0.0.1 // do not remove until https://github.com/mattn/go-ieproxy/issues/45 is fixed
+replace github.com/minio/madmin-go/v2 v2.0.6 => ../madmin-go
```

Pro tip:
```suggestion
replace github.com/minio/madmin-go/v2 v2.0.6 => github.com/Praveenrajmani/madmin-go 8c61990845d69891b985213fc1157a8e80188025
```

This will make it buildable using the commit on your fork after you run `go mod tidy`.